index.html

```diff
@@ -241,5 +241,35 @@ <h2>Examples</h2>
         ...
       </pre>
     </section>
+    <section class="informative">
+      <h2>Privacy and Security Considerations</h2>
+      <p>
+        The mechanisms introduced by this specification are only available to an application which
+        has called {{MediaDevices/getDisplayMedia()}}, and where the user selected to capture some
+        [=display surface=] <var>S</var>. This means that the capturing application already has
+        access to all of the pixels visible on <var>S</var>.
+      </p>
+      <p>
+        If the capturing application so chooses, it can include the cursor among those pixels by
+        specifying the {{CursorCaptureConstraint}} {{CursorCaptureConstraint/"always"}}. This can
+        either be done when calling {{MediaDevices/getDisplayMedia()}}, or through
+        {{MediaStreamTrack/applyConstraints()}}. If {{MediaStreamTrack/applyConstraints()}} is used,
+        the cursor constraint might even be applied to a clone of the original track.
+      </p>
```

Comment:
In the other PR, I'm mentioning that constraints don't affect how {{MediaDevices/getDisplayMedia()}}'s permission request is presented to the user, otherwise the constraint passed is still be relevant. Do you mean here it's possible e.g. to clone a track originally with a "never" constraint, and to apply the "always" constraint instead?

Reply:
I am saying that the cursor is **available** to apps, and that this is the important part; whether they choose to access the cursor is less relevant, I claim.